wandb/interface/interface.py

```diff
@@ -515,10 +519,16 @@ def _send_exit_sync(self, exit_data, timeout=None):
         assert result.exit_result
         return result.exit_result
 
-    def send_defer(self, uuid):
+    def send_poll_exit_sync(self, timeout=None):
+        poll_request = wandb_internal_pb2.PollExitRequest()
+        rec = self._make_request(poll_exit=poll_request)
+        rec.control.local = True
```

Comment:
Remind me what this does?  Should I be setting `control.local` on my login or pause / resume requests?

Reply:
@vanpelt, yes, i am going to move setting this into _make_request, so that will fix the other cases.
The idea is it doesnt need to be sycned remotely so it doesnt go into the persistent store, (which is the case for all "messages")